### PR TITLE
Policy Refresh Q2FY24

### PR DESCRIPTION
**Pull request by Springstone**
## Overview/Summary

The Policy Refresh (slightly delayed) for Q2FY24 which includes a number of enhancements and transition from MMA to AMA. Please review What's New for details.

## This PR fixes/adds/changes/removes

1. See What's New

### Breaking Changes

1. None

## Testing Evidence

Please provide any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

#### Azure Public

[Deploy To Azure](https://portal.azure.com/#view/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FEnterprise-Scale%2Fpolicy-refresh-q2fy24%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FEnterprise-Scale%2Fpolicy-refresh-q2fy24%2FeslzArm%2Feslz-portal.json)

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Ensured [contribution guidance](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Contribution-Guide) is followed.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Just a few bits

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -42,6 +38,26 @@ This article will be updated as and when changes are made to the above and anyth
 
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
+### 🔃 Policy Refresh Q2 FY24
+
+Yes, the Q2 Policy Refresh has been delayed due to a light past quarter and some very important initiatives that we feel had to make it into this refresh.
+
+#### Policy
```
[the remark this answers is not in the document]
Comment (arjenhuitema) [reply docs/wiki/Whats-new.md:45]: I'm updating this page with an AMA update.

File: docs/wiki/ALZ-Policies-FAQ.md [hunk 2]
```diff
@@ -14,10 +14,23 @@ Check back here for updates, and be sure to bookmark [What's New](https://aka.ms
 
 To view the current list of GitHub issues related to diagnostic settings, please see [this link](https://github.com/Azure/Enterprise-Scale/labels/Area:%20Diagnostic%20Settings).
 
+> **UPDATE** New built-in Diagnostic Settings policies and initiatives will be landing in early CY2024. As a heads-up we will begin deprecating all our custom diagnostic settings policies, and changing our default assignment to leverage the associated built-in initiative for Log Analytics (as the target) - additional options will include targeting Event Hubs or Storage accounts.
+
 ### Azure Monitor Agent (May 2023)
```
[the remark this answers is not in the document]
Comment (arjenhuitema) [reply docs/wiki/ALZ-Policies-FAQ.md:19]: Paul will be updating the FAQ statements. Not sure which particular statement, lets wait for his comment.

[the remark this answers is not in the document]
Comment (paulgrimley) [reply docs/wiki/ALZ-Policies-FAQ.md:19]: I wasnt planning to update this page was planning to update the existing MMA update

[the remark this answers is not in the document]
Comment (jtracey93) [reply docs/wiki/ALZ-Policies-FAQ.md:19]: can you please update this one @paulgrimley in your PR, or update this page to link to yours for a single message.

**Review by jtracey93 — APPROVED** [review]

**Pull request merged**